src/routes/safes/safes.service.ts

```diff
@@ -176,6 +179,20 @@ export class SafesService {
         );
       }),
     );
+
+    const safeOverviews: Array<SafeOverview> = [];
+
+    for (const safeOverview of settledOverviews) {
+      if (safeOverview.status === 'rejected') {
+        this.loggingService.error(
```

Comment:
I would change this to `warn` level as I see this message as something that we need to be aware of but doesn't stop the endpoint from working.

Reply:
Changed in 15d1d5fd456a45d446a767856d230dd477705079.